pkg/admission/certgenerator/certgenerator.go

```diff
@@ -24,8 +24,8 @@ type Certs struct {
 	CACert []byte
 }
```

Comment:
`Certs` is giving an impression of multiple certificates and it is not.

some options.`Cert` or `Certificate` or `CertArtifacts` or `CertContents`